login-fire-form.html

```diff
@@ -365,7 +365,13 @@
         this.email = null;
         this.password = null;
       } else {
-        this.set('_lastError', e.detail.message);
+        // Localize firebase error if possible
+        var errorMessage = this.localize('lf-error-'+e.detail.code);
+        if(!errorMessage) {
+          this.set('_lastError', e.detail.message);
+        } else {
+          this.set('_lastError', errorMessage);
+        }
```

Comment:
I guess we could replace the `if-else` block by `this.set('_lastError', errorMessage || e.detail.message);` but it's OK.